docs/standalone.md

```diff
@@ -227,3 +227,7 @@ and you will get the response like this:
  }
 
 ```
+By default there is no compression for the response of the rest apis. If you want to reduce the consumption of network transmission, you can set header `Accept-Encoding` to `gzip`. The corresponding `curl` command will be like:
```

Comment:
@jimmidyson Your way of description looks more formal and better. I have refreshed the doc, please have a look, bro.